frontend/package.json

```diff
@@ -47,6 +47,8 @@
 		"@tanstack/react-virtual": "3.11.2",
 		"@uiw/react-md-editor": "3.23.5",
 		"@visx/group": "3.3.0",
+		"@visx/hierarchy": "3.12.0",
```

Comment:
Added for the treemap implementation

Reply:
Yes I did. We need it to build a hierarchical structure here for which the treemap component is necessary. And it's already optimized to work with large datasets like ours